etc/module.xml

```diff
@@ -1,20 +1,20 @@
-<?xml version="1.0"?>
-<!--
-/**
- * Copyright © 2018 Cardknox Development Inc. All rights reserved.
- * See LICENSE for license details.
- */
--->
-
-<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <!-- Version is saved in the database, cannot be downgraded without changing the data entry -->
-    <module name="CardknoxDevelopment_Cardknox" setup_version="2.0.14">
-        <sequence>
-            <module name="Magento_Sales" />
-            <module name="Magento_Payment" />
-            <module name="Magento_Directory" />
-            <module name="Magento_Config" />
-            <module name="Magento_Vault"/>
-        </sequence>
-    </module>
-</config>
+<?xml version="1.0"?>

```

Comment:
Are these all whitespace changes? Are they necessary?

Reply:
I download cardknox extension from the marketplace and then upgrading Magento v2.4.5 with GooglePay, So all lines are showing as modified. I just updated **setup_version=2.0.15** in module.xml

Reply:
I think the files in the marketplace use Unix line endings, but these files use windows line endings.
Will this make a difference to magento/php?

Reply:
No, There are no issue with Unix line endings ans windows line endings.